Scripts/Python/QueryGroupManagementOmeREDFISH.py

```diff
@@ -0,0 +1,706 @@
+#
+# QueryGroupManagementOmeREDFISH. Python script using Redfish API with OEM extension to manage Open Manage Enterprise Query Groups
+#
+# _author_ = Texas Roemer <Texas_RoemerDell.com>
+# _version_ = 1.0
+#
+# Copyright (c) 2020, Dell, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
```

Comment:
All software on this site is governed by Apache License v 2.0 - would you be willing to release the script under that license term and rework this?